src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import { FC } from 'react';
+import { User } from '../../types/types';
+
+type Props = {
```

Comment:
You've already started nicely to export typing to separate files, so it would be cool to continue here as well, and export the type for the `UserInfo` component to the types.tsx file and give this type a more convenient name (For example: `UserInfoType`).